src/components/shared/Error/Message.tsx

```diff
@@ -25,10 +35,17 @@ function Message({ error }: Props) {
         (typeof error === 'object' && error.hasOwnProperty('code')) ||
         failedAfterRetry;
 
+    // There is a small chance this can happen so adding custom event to track it
+    // Happened before when journal data hook wasn't catching errors
+    const id = error.message ?? FALLBACK;
+    if (id === FALLBACK) {
+        logRocketEvent(CustomEvents.ERROR_MISSING_MESSAGE);
+    }
```

Comment:
Make this super safe. This spot should no longer get hit - but in case making sure we have an event just for it. 

We _could_ figure this out with the event/logging above but I want to be able to search for this and not have to manually review sessions.